contrib/test_jenkins.sh

```diff
@@ -180,6 +180,47 @@ get_active_ib_devices() {
 	done
 }
 
+#
+# Get list of active IP interfaces
+#
+get_active_ip_ifaces() {
+	echo $(ip addr | awk '/state UP/ {print $2}' | sed s/://)
+}
+
+#
+# Get IP addr for a given IP iface
+# Argument is the IP iface
+#
+get_ifaddr() {
+	iface=$1
+	echo $(ip addr show ${iface} | awk '/inet /{print $2}' | awk -F '/' '{print $1}')
+}
+
+get_ipoib_address() {
```

Comment:
does it work for roce as well, or just ipoib?

Reply:
yes, it works for roce as well

is `get_rdma_device_ip_addr` better?

Reply:
yes

Reply:
done